### PR TITLE
RHDEVDOCS-5955: Replacing the {product-title} attribute with the {OCP} attribute

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
**Version(s):** GitOps 1.9, GitOps 1.10, GitOps 1.11, GitOps 1.12

**Issue:** https://issues.redhat.com/browse/RHDEVDOCS-5955

**Link to docs preview:**: 
https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/troubleshooting_gitops_issues/auto-reboot-during-argo-cd-sync-with-machine-configurations.html

https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-configuring-sscsi-driver-to-mount-secrets-from-aws-secrets-manager_managing-secrets-securely-using-sscsid-with-gitops

https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/troubleshooting_gitops_issues/auto-reboot-during-argo-cd-sync-with-machine-configurations

https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/observability/monitoring/health-information-for-resources-deployment#go-settings-for-environment-labels-and-annotations_health-information-for-resources-deployment

https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-storing-aws-secret-manager-resources-in-gitops-repository_managing-secrets-securely-using-sscsid-with-gitops

SME review: @reginapizza 
QE review: @varshab1210 
Internal Peer review: @eromanova97
Peer review:

**QE review:**
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/gitops-keycloak-identity-brokering-with-openshift-oauthclient.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 // *
 
 [id="keycloak-identity-brokering-with-openshift-oauthclient_{context}"]
-= Keycloak Identity Brokering with {product-title}
```
Comment (varshab1210) [inline modules/gitops-keycloak-identity-brokering-with-openshift-oauthclient.adoc:6]: What was {product-title} earlier?

**Comment by varshab1210** [comment]
LGTM

**Review by eromanova97 — COMMENTED** [review]
Found one more instance here: https://docs.openshift.com/gitops/1.12/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.html#gitops-storing-aws-secret-manager-resources-in-gitops-repository_managing-secrets-securely-using-sscsid-with-gitops

There is "OpenShift Container Platform" instead of {OCP} at the beginning.

Otherwise LGTM

**Review by reginaelyse — APPROVED** [review]

**Comment by bburt-rh** [comment]
> > @Dhruv-Soni11 Please provide direct links to each of the files you changed. As a peer reviewer, I need to verify that the variables are rendering properly. I cannot approve the PR without seeing all the changes in a preview.
> 
> Apologies @bburt-rh, here are all the links I could find, some changes are made in the openshift doc repository, for which the preview does not generate a clickable link.
> 
> https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-configuring-sscsi-driver-to-mount-secrets-from-aws-secrets-manager_managing-secrets-securely-using-sscsid-with-gitops
> 
> https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/troubleshooting_gitops_issues/auto-reboot-during-argo-cd-sync-with-machine-configurations
> 
> https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/observability/monitoring/health-information-for-resources-deployment#go-settings-for-environment-labels-and-annotations_health-information-for-resources-deployment
> 
> https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-storing-aws-secret-manager-resources-in-gitops-repository_managing-secrets-securely-using-sscsid-with-gitops

@Dhruv-Soni11 - I think maybe you meant to tag @mburke5678?

**Comment by Dhruv-Soni11** [comment]
> > > @Dhruv-Soni11 Please provide direct links to each of the files you changed. As a peer reviewer, I need to verify that the variables are rendering properly. I cannot approve the PR without seeing all the changes in a preview.
> > 
> > 
> > Apologies @bburt-rh, here are all the links I could find, some changes are made in the openshift doc repository, for which the preview does not generate a clickable link.
> > https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-configuring-sscsi-driver-to-mount-secrets-from-aws-secrets-manager_managing-secrets-securely-using-sscsid-with-gitops
> > https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/troubleshooting_gitops_issues/auto-reboot-during-argo-cd-sync-with-machine-configurations
> > https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/observability/monitoring/health-information-for-resources-deployment#go-settings-for-environment-labels-and-annotations_health-information-for-resources-deployment
> > https://74871--ocpdocs-pr.netlify.app/openshift-gitops/latest/securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops#gitops-storing-aws-secret-manager-resources-in-gitops-repository_managing-secrets-securely-using-sscsid-with-gitops
> 
> @Dhruv-Soni11 - I think maybe you meant to tag @mburke5678?

Yes, I did. @mburke5678 - Please find attached the links as you requested.

File: modules/gitops-registering-an-additional-oauth-client.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 [id="registering-an-additional-oauth-client_{context}"]
 = Registering an additional an OAuth client
```
Comment (skopacz1) [inline modules/gitops-registering-an-additional-oauth-client.adoc:6]: This is out of scope so I won't stop the merging over this, but it seems like this heading has a typo:
```suggestion
= Registering an additional OAuth client
```

File: modules/gitops-registering-an-oauth-client.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 [id="registering-an-additional-oauth-client_{context}"]
 = Registering an additional an OAuth client
```
Comment (skopacz1) [inline modules/gitops-registering-an-oauth-client.adoc:6]: Same comment as above:
```suggestion
= Registering an additional OAuth client
```

**Review by skopacz1 — COMMENTED** [review]
Left a few out of scope comments for you to consider fixing in a separate PR, but otherwise LGTM! I'll proceed to merge

**Comment by Dhruv-Soni11** [comment]
Hi @skopacz1 - Please find the cherry-pick links for GitOps 1.10 and GitOps 1.11 branches added here:

GitOps 1.10 - https://github.com/openshift/openshift-docs/pull/75753
GitOps 1.11 - https://github.com/openshift/openshift-docs/pull/75754

**Pull request merged**